index.html

```diff
@@ -1,137 +1,37 @@
-          <td rowspan="2">Defines a "header cell" / "cell" in a table</td>
-        </tr>
-        <tr>
-          <td>&lt;td&gt;</td>
-        </tr>
-        <tr>
-          <td colspan="2">Rarely used</td>
-        </tr>
-        <tr>
-          <td>&lt;caption&gt;</td>
-          <td>Defines a table caption</td>
-        </tr>
-        <tr>
-          <td>&lt;thead&gt;</td>
-          <td>Groups the header content in a table</td>
-        </tr>
-        <tr>
-          <td>&lt;tbody&gt;</td>
-          <td>Groups the body content in a table</td>
-        </tr>
-        <tr>
-          <td>&lt;tfoot&gt;</td>
-          <td>Groups the footer content in a table</td>
-        </tr>
-      </tbody>
-    </table>
-    <br />
-    <form action="/" method="get">
-      <fieldset>
-        <legend>My first form</legend>
-        <label for="name">Name</label><br />
-        <input id="name" name="name" placeholder="Name" required /><br />
-        <label for="email">Email</label><br />
-        <input id="email" type="email" name="email" required /><br />
-        <label for="password">Password</label><br />
-        <input id="password" type="password" name="password" required /><br />
-        <label for="level">Your programming level</label><br />
-        <input id="level" type="range" name="level" min="1" max="10" /> <br />
-        <p>What does HTML stand for?</p>
-        <input id="answer1" type="checkbox" name="HTML" value="off" />
-        <label for="answer1">Hyperlinks and Text Markup Language</label><br />
-        <input id="answer2" type="checkbox" name="HTML" value="off" />
-        <label for="answer2">Home Tool Markup Language</label><br />
-        <input id="answer3" type="checkbox" name="HTML" value="on" />
-        <label for="answer3">Hyper Text Markup Language</label><br />
-        <p>What does HTTP stand for?</p>
-        <input id="1" type="radio" name="HTTP" value="off" />
-        <label for="1">Hyper Text Translate Protocol</label><br />
-        <input id="2" type="radio" name="HTTP" value="off" />
-        <label for="2">Hyper Text Transmission Protocol</label><br />
-        <input id="3" type="radio" name="HTTP" value="on" />
-        <label for="3">Hyper Text Transfer Protocol</label><br />
-        <label for="comment">Comment</label><br />
-        <textarea id="comment" name="comment" maxlength="500">Comment</textarea
-        ><br />
-        <label for="HTML">What does HTML stand for?</label><br />
-        <select id="HTML" name="select_html">
-          <option disabled selected>Your answer</option>
-          <option value="1">Hyperlinks and Text Markup Language</option>
-          <option value="2">Home Tool Markup Language</option>
-          <option value="3">Hyper Text Markup Language</option>
-        </select>
-        <p><button type="submit">Send</button></p>
-      </fieldset>
-    </form>
+    <h2 class="offer">What do we offer</h2>
+    <hr class="line" />
+    <div class="container">
```

Comment:
**Just recommendation**, it's extra info for now
Don't add wrappers if you can escape them
for example you could use only` <figure>` here

or if you want to add wrapper, it's better to use `ul `and` li`, because thihs is a list of elements